boot/boot.go

```diff
@@ -403,19 +416,40 @@ func makeBootable20(model *asserts.Model, rootdir string, bootWith *BootableSet,
 	return nil
 }
 
+func makeBootable20RunMode(model *asserts.Model, rootdir string, bootWith *BootableSet) error {
+	// XXX: move to dirs ?
+	runMnt := filepath.Join(rootdir, "/run/mnt/")
+
+	// TODO:UC20:
+	// - create grub.cfg instead of using the gadget one
+	// - extract kernel
+	// - write modeenv
+
+	// update recovery grubs grubenv to indicate that we transition
+	// to run mode now
+	bl, err := bootloader.Find(filepath.Join(runMnt, "ubuntu-seed"), nil)
```

Comment:
don't we need a bootloader with opts.Recovery = true here ?

Reply:
Yes, fixed, silly me! Sorry